### DISCUSSION TITLE
Set axios default values in newer versions?

### DISCUSSION THREAD

**Discussion by sagerio** [directus/directus#4924] [Archive]
In previous versions (< 9.0.0-rc.55) of directus, one could set default values to axios like this (Typescript):

`this.directus.axios.defaults.headers["Authorization"] = "Bearer " + environment.Directus.Token;`

How this could be done in versions > 9.0.0-rc.55?

Thanks a lot!

**Comment by rijkvanzanten** [comment]
@WoLfulus ?

**Comment by WoLfulus** [comment]
if you're trying to do exactly what you are doing in the example (Authorization header) I'd suggest using `await sdk.auth.static(environment.Directus.Token)` instead. If it's for something else, you can get to the axios instance through the transporter:

The "ugly" way to get there is 

```
(sdk.transport as AxiosTransport).axios.defaults.headers
(sdk.transport as AxiosTransport).axios.interceptors
```

If you're instantiating the `AxiosTransporter` yourself and passing it to the SDK constructor, you should already have an instance of `AxiosTransport`, so you don't need the cast

**Reply by sagerio** [reply]
OK I'll try, thanks a lot!

**Reply by sagerio** [reply]
For the records: `sdk.auth.static(<token>)` works like a charm! :thumbsup:

**Reply by sagerio** [reply]
Sorry for pulling this up again, but had something changed in the latest rc59?
Neither one of the above described ways work for me anymore via gthe SDK, the response is always
`401 (Unauthorized) Error: Invalid user credentials. `
But calling via the WebAPI everything is fine, so the token is correct...

**Reply by sagerio** [reply]
I figured it out, the old token was still saved in the session storage and therefore even outlived a browser restart and got still loaded although I set the a new value in code :persevere:
